plaso/parsers/sqlite_plugins/chrome_history.py

```diff
@@ -15,8 +15,14 @@ class ChromeHistoryFileDownloadedEventData(events.EventData):
   """Chrome History file downloaded event data.
 
   Attributes:
+    danger_type (int): enum indicating if the download was dangerous.
```

Comment:
Please don't abbreviate and write enumeration (also see https://github.com/log2timeline/l2tdocs/blob/main/process/Style-guide.md#docstrings)

IMHO you can leave out the fact that this is an enum (same applies to the other values)

Reply:
maybe:
```
assessment of the danger of the downloaded content.
```

Maybe indicate what this assessment is based on? Safe Browsing?

Reply:
All these descriptions were copied from the linked Chromium source files. I generally try to alter the "source of truth" as little as possible, but I can change to match plaso's style.

Reply:
Changed